api/DataChannelMessageParser.cpp

```diff
@@ -8,65 +7,69 @@ namespace DataChannelMessageParser
 {
 namespace
 {
-bool isLegacyApi(const nlohmann::json& messageJson)
+bool isLegacyApi(const utils::SimpleJson& messageJson)
 {
-    return messageJson.find("colibriClass") != messageJson.end();
+    return messageJson.exists("colibriClass");
 }
 } // namespace
 
-bool isPinnedEndpointsChanged(const nlohmann::json& messageJson)
+bool isPinnedEndpointsChanged(const utils::SimpleJson& messageJson)
 {
+    char messageType[200];
     if (isLegacyApi(messageJson))
     {
-        return legacyapi::DataChannelMessageParser::isPinnedEndpointsChanged(messageJson);
+        messageJson["colibriClass"].getString(messageType);
```

Comment:
I think it would be nice if `getString()` could return an `utils::span<char>` so we could make the comparison with zero memory copies